### PR TITLE
[CLA] Signature for Juan Manuel Rosero Puente

### PR DESCRIPTION
**Pull request by odoo-group-ecuador**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by odoo-group-ecuador** [comment]
PR previous l10n Ecuador proposal
@qdp-odoo

**Comment by odoo-group-ecuador** [comment]
PR previous l10n Ecuador proposal
Hello @mart-e @qdp-odoo @antonylesuisse

File: doc/cla/individual/juanmanuel.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Ecuador, 31 January 2017
+
+I hereby agree to the terms of the Odoo Individual Contributor License Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this declaration.
+
+Signed,
+
+Juan Manuel Rosero Puente snowblow07@gmail.com https://www.github.com/snowblow/
```
Comment (mart-e) [inline doc/cla/individual/juanmanuel.md]: are you using the @snowblow account or @odoo-group-ecuador?

**Comment by mart-e** [comment]
Hello,

Thank you for the signature, however, I see that you are signing an individual CLA with the account @snowblow.
I am not sure to understand what you are saying about the promotion.
If you wish to commit as an organistation (which I guess @odoo-group-ecuador represent), you can sign a corporate CLA.

**Comment by odoo-group-ecuador** [comment]
Footnote was because somebody here in Ecuador took advantage of our first pull request to promote subscriptions to OCA and as independent developers we need to clarify this point because as result of that some people left group, about version i will pull next proposal only from myself,

Please give me your opinion because I want start work ASAP.

**Comment by mart-e** [comment]
> Please give me your opinion

but I already gave you my opinion

- do not create an individual signature with user A if you are signing for user B
- change the signature filename to match to correct user
- are you going to make contributions only in master or also in stable versions?

**Pull request closed without merging**